### PR TITLE
1) Add a feature to disable a node. 2) Reduce around 400kB in the transpiled code. 3) Adding support to React node type (string or jsx tags) to the node label.

### PR DESCRIPTION
**Pull request by lucas-issa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,8 +110,9 @@ Individual nodes within the `nodes` property can have the following structure:
 
 | Property    | Type   | Description                     |
 | ----------- | ------ | ------------------------------- |
-| `label`     | string | **Required**. The node's label. |
+| `label`     | mixed  | **Required**. The node's label. |
 | `value`     | mixed  | **Required**. The node's value. |
 | `children`  | array  | An array of child nodes.        |
 | `className` | string | A className to add to the node. |
 | `icon`      | mixed  | A custom icon for the node.     |
+| `disabled`  | bool   | Disable node.                   |
```
Comment (jakezatecky) [inline README.md]: I would prefer `disabled` to appear above `icon` and below `className` to be alphabetically sorted below the required properties.

File: src/js/CheckboxTree.js [hunk 2]
```diff
@@ -190,13 +190,14 @@ class CheckboxTree extends React.Component {
             const key = `${node.value}`;
             const checked = this.getCheckState(node, noCascade);
             const children = this.renderChildNodes(node);
+            const nodeDisabled = !!(disabled || node.disabled);
```
Comment (jakezatecky) [inline src/js/CheckboxTree.js]: Is there a reason the double negation is being used?

File: README.md (hunk 1 above)
Comment (lucas-issa) [inline README.md]: Ok. No problem.

File: src/js/CheckboxTree.js (hunk 2 above)
Comment (lucas-issa) [inline src/js/CheckboxTree.js]: Yes. The disabled property on TreeNode isRequired. The double negation will transform undefined to false.

**Comment by samuel-gaunt** [comment]
Any update on this?

**Comment by jakezatecky** [comment]
Soon. Might ship out a new release later this week.

**Pull request merged**